pyrolite/plot/templates/__init__.py

```diff
@@ -27,4 +28,5 @@
     "FeldsparTernary",
     "SpinelFeBivariate",
     "SpinelTrivalentTernary",
+    "sandstones",
```

Comment:
Rather than `"sandstones"` here, I'd suggest using the diagram names `"Pettijohn", "Herron"`.